api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -428,3 +414,128 @@ def comment_incident(finance_incident_id: int) -> utils.BackofficeResponse:
+    )
+    bookings_venue = (
+        finance_incident.booking_finance_incidents[0].collectiveBooking.venue
+        if finance_incident.booking_finance_incidents[0].collectiveBooking
+        else finance_incident.booking_finance_incidents[0].booking.venue
+    )
+
+    if bookings_venue.current_reimbursement_point_id:
+        current_reimbursement_point = offerer_models.Venue.query.get(bookings_venue.current_reimbursement_point_id)
+    else:
+        current_reimbursement_point = bookings_venue
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=empty_forms.EmptyForm(),
+        dst=url_for(
+            "backoffice_v3_web.finance_incident.validate_finance_incident", finance_incident_id=finance_incident_id
+        ),
+        div_id=f"finance-incident-validation-modal-{finance_incident_id}",
+        title="Valider l'incident",
+        button_text="Confirmer",
+        information=f"Vous allez valider un incident de {incident_total_amount_euros} € "
+        f"sur le point de remboursement {current_reimbursement_point.name}. Voulez-vous continuer ?",
+    )
+
+
+@finance_incident_blueprint.route("/incident/<int:finance_incident_id>/validate", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def validate_finance_incident(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = finance_models.FinanceIncident.query.options(
+        # laad booking beneficiary
+        sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents).joinedload(
+            finance_models.BookingFinanceIncident.beneficiary
+        ),
+        # load booking pricing
+        sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents)
+        .joinedload(finance_models.BookingFinanceIncident.booking)
+        .joinedload(bookings_models.Booking.pricings)
+        .load_only(finance_models.Pricing.amount),
+        # load collective booking pricing
+        sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents)
+        .joinedload(finance_models.BookingFinanceIncident.collectiveBooking)
+        .joinedload(educational_models.CollectiveBooking.pricings)
+        .load_only(finance_models.Pricing.amount),
+        # load incident venue
+        sa.orm.joinedload(finance_models.FinanceIncident.venue),
+    ).get_or_404(finance_incident_id)
+
+    if finance_incident.status != finance_models.IncidentStatus.CREATED:
+        flash("L'incident ne peut être validé que s'il est au statut 'créé'.", "warning")
+        return render_finance_incident(finance_incident)
+
+    _validate_finance_incident(finance_incident)
+
+    flash("L'incident a été validé avec succès.", "success")
+    return render_finance_incident(finance_incident)
+
+
+def _validate_finance_incident(finance_incident: finance_models.FinanceIncident) -> None:
+    finance_incident_link = f"{settings.API_URL}/finance/incident/{finance_incident.id}/"
```

Comment:
Ne voudrais-tu pas l'URL du backoffice plutôt que celle de l'API backend ?
```suggestion
    finance_incident_link = f"{settings.BACKOFFICE_URL}/finance/incident/{finance_incident.id}/"
```

Reply:
Je vais voir pour corriger ca effectivement, j'ai l'impression qu'en local BACKOFFICE_URL n'est pas set. Je vais checker aussi

Reply:
Tu peux l'ajouter dans `.env.development`.
Mais avec `API_URL`, sur testing, staging ou en prod, ton lien ne marchera pas :)

Reply:
Changement de stratégie concernant le commentaire -> fermeture du thread